frontend/scripts/react-components/profile/profile-widgets/top-consumers-widget.component.jsx

```diff
@@ -3,110 +3,123 @@ import PropTypes from 'prop-types';
 import Widget from 'react-components/widgets/widget.component';
 import MiniSankey from 'react-components/profile/profile-components/mini-sankey/mini-sankey.component';
 import { withTranslation } from 'react-components/nav/locale-selector/with-translation.hoc';
-import {
-  getSummaryEndpoint,
-  GET_PLACE_TOP_CONSUMER_ACTORS,
-  GET_PLACE_TOP_CONSUMER_COUNTRIES,
-  GET_COUNTRY_TOP_CONSUMER_COUNTRIES
-} from 'utils/getURLFromParams';
+import { getSummaryEndpoint } from 'utils/getURLFromParams';
 import ShrinkingSpinner from 'react-components/shared/shrinking-spinner/shrinking-spinner.component';
 import Heading from 'react-components/shared/heading/heading.component';
 import ProfileTitle from 'react-components/profile/profile-components/profile-title.component';
 
 const TranslatedMiniSankey = withTranslation(MiniSankey);
 
-function TopConsumersWidget (props) {
-  const { onLinkClick, year, nodeId, contextId, type, testId, title, commodityName, profileType } = props;
-  const handleLinkClick = (_, { query }) => {
+class TopConsumersWidget extends React.PureComponent {
```

Comment:
changed to class based component so we are consistent with the other widgets, this might change when we have time to refactor the widgets.

Reply:
Actually the idea was to slowly change the classes to functions and hooks whenever we edit the code. But it's ok